templates/event.php

```diff
@@ -58,9 +58,9 @@
 			</thead>
 			<tbody>
 			<?php /** @var $row Stats_Row */ ?>
-			<?php foreach ( $event_stats->rows() as $locale_ => $row ) : ?>
+			<?php foreach ( $event_stats->rows() as $_locale => $row ) : ?>
 			<tr>
-				<td><?php echo esc_html( $locale_ ); ?></td>
+				<td title="<?php echo esc_html( $_locale ); ?> "><a href="<?php echo esc_url( gp_url( '/locale/' . $row->language->slug ) ); ?>"><?php echo esc_html( $row->language->english_name ); ?></a></td>
```

Comment:
Local GlotPress uses `'/languages/'` else a 404 is returned when the locale link is clicked.